src/components/Editor.jsx

```diff
@@ -1,47 +1,65 @@
-//import { useState, useCallback, useEffect } from "react";
-import { FormLayout, TextField } from "@shopify/polaris";
+import React, { useState } from "react";
+import { FormLayout } from "@shopify/polaris";
 import "./Editor.css";
 
 function Editor({ jsonCode, onJsonChange, updatedJson, error }) {
-  //const [jsonCode, setJsonCode] = useState("");
-  //const [updatedJsonCode, setUpdatedJsonCode] = useState("");
-  // Convert the updatedJson string to an object
-  /* const parsedUpdatedJson = JSON.parse(updatedJson);
+  const [text, setText] = useState("");
+
+  const handleTextChange = (event) => {
+    const inputText = event.target.value;
+
+
+    if (inputText.endsWith("{")) {
+      const newText = inputText + "\n  \" \"\n}";
+      setText(newText);
+      onJsonChange(newText);
+
+      // Set the cursor position between the braces
+      const cursorPosition = newText.length - 5;
+      event.target.setSelectionRange(cursorPosition, cursorPosition);
+    } else if (inputText.endsWith(`"`)) {
+      const newText = inputText + `"`;
+      setText(newText);
+      onJsonChange(newText);
+    } else {
+      setText(inputText);
+      onJsonChange(inputText);
+    }
+  };
+
+
 
-  // Convert the parsed object to a formatted JSON string
-  const formattedJsonString = JSON.stringify(parsedUpdatedJson, null, 2); */
 
   return (
     <FormLayout>
       <style>{`.Polaris-TextField__Resizer {display: none;}`}</style>
       <div className="editor-wrapper">
         <div className="input-field-wrapper">
-          <TextField
-            wrap="true"
+          <p> Paste JSON Code</p>
+          <textarea
+            className="Polaris-TextField__Input Polaris-TextField__Input--multiline textarea"
```

Comment:
Hey @abhishekjani08 a quick comment about this, as I pointed out [earlier in this conversation](https://github.com/dear-digital/linter/pull/17#pullrequestreview-1569548785), can you keep the `TextField` component in use without interjecting the classnames `Polaris-TextField__Input Polaris-TextField__Input--multiline` directly into the javascript, as this goes against the standard React practices.